quickstarts/router-ext/README.md

```diff
@@ -0,0 +1,258 @@
+## Red Hat Process Automation Manager SmartRouter Extension Example
+
+Note: this is the development branch, the target images might not be available here, instead you can look at the [release branch](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.9.x/quickstarts/router-ext)
+
+Use this quick start guide with the [RHPAM SmartRouter](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/master/smartrouter) image.
+
+The SmartRouter extensions are used to extend the Smart Router functionalities to make it to adapt to your needs.
+This example provides a very simple custom Kie Container resolver based on its version and how you can add this
+extension to a custom Smart Router image.
+
+## Using the example SmartRouter extension
+
+Build the source code with Maven:
+
+```bash
+$ mvn clean package
+```
+
+The build process will generate the following jar: `target/router-ext-0.0.1-SNAPSHOT.jar` 
+
+After building the Smart Router extension, you can prepare the environment to extend the `registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.9.0` image 
+and customize it, to include the recently built Smart Router extension.
+
+If you don't have the Smart Router image on your local registry, pull it:
+
+```bash
+$ docker pull registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.9.0
+```
+
+Create a directory for building the custom Smart Router Image. For this example we use. For this example I'll be using 
```

Comment:
```suggestion
Create a directory for building the custom Smart Router Image. For this example we use 
```

Reply:
" For this example I'll be using the tmp directory." sounds better, no?
as it is a personal choice on where to create this directory.